homeassistant/core.py

```diff
@@ -818,33 +887,32 @@ async def async_stop(self, exit_code: int = 0, *, force: bool = False) -> None:
 
         self.exit_code = exit_code
 
-        # stage 1
         self.state = CoreState.stopping
         self.bus.async_fire(EVENT_HOMEASSISTANT_STOP)
         try:
-            async with self.timeout.async_timeout(STAGE_1_SHUTDOWN_TIMEOUT):
+            async with self.timeout.async_timeout(STOP_STAGE_SHUTDOWN_TIMEOUT):
                 await self.async_block_till_done()
         except asyncio.TimeoutError:
             _LOGGER.warning(
-                "Timed out waiting for shutdown stage 1 to complete, the shutdown will"
+                "Timed out waiting for integrations to stop, the shutdown will"
                 " continue"
             )
-            self._async_log_running_tasks(1)
+            self._async_log_running_tasks(2)
```

Comment:
Since we renamed the stages from `1`, `2`, etc, `1`,`2`, should be named as well so they match the stage name

Reply:
Makes sense, fixed